DelegateFun/Sorter/SortUtility.cs

```diff
@@ -1,10 +1,35 @@
-﻿using System;
+using System;
 
 namespace Sorter
 {
     public class SortUtility
     {
         // Sort method should be implemented here
         // It should accept an int[] and a delegate you define that performs the actual comparison
+        public delegate bool IsLowerOrHigher(int x, int y);
+
+        public int[] SelectionSort(int[] ara, IsLowerOrHigher isLowerOrHigher)
+        {
+            if (ara == null)
+                throw new ArgumentNullException("The Integer Array is Null.");
+            else if (isLowerOrHigher == null)

```

Comment:
```suggestion
            if (isLowerOrHigher == null)
```
Good call - I forgot to do this on mine. However you don't need and if-else here, you can just use two if's.